substrate/frame/staking-async/src/pallet/mod.rs

```diff
@@ -2162,6 +2200,7 @@ pub mod pallet {
 			config_op_exp!(ChillThreshold<T>, chill_threshold);
 			config_op_exp!(MinCommission<T>, min_commission);
 			config_op_exp!(MaxStakedRewards<T>, max_staked_rewards);
+			config_op_exp!(UnbondingQueueParams<T>, unbonding_queue_params);
```

Comment:
Should we have some integrity checks here? Such as lower bound should never exceed `UnbondingDuration`.